### PR TITLE
Fix NPE in GetDatabase

### PR DESCRIPTION
**Pull request by yuzhu**
Location and Comment are optional fields, so we must treat them as such..
Apparently, so is the Owner and OwnerType

### CODE REVIEW

**Comment by yuzhu** [comment]
jenkins, test this please

File: table/server/common/src/main/java/alluxio/master/table/DatabaseInfo.java [hunk 1]
```diff
@@ -26,7 +26,9 @@
 public class DatabaseInfo {
   @Nullable
   private final String mLocation;
+  @Nullable
   private final String mOwnerName;
+  @Nullable
   private final PrincipalType mOwnerType;
```
Comment (ZacBlanco) [inline table/server/common/src/main/java/alluxio/master/table/DatabaseInfo.java]: should the default values in the constructor continue to be `""` and `PrincipalType.USER`? or should they be `null`? what is the difference between `""` and `null` for `mOwnerName`?

SImilar questions for the location and comment

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Comment by yuzhu** [comment]
alluxio-bot, cherry-pick this to branch-2.1 please

**Pull request merged**